.php_cs

```diff
@@ -12,7 +12,7 @@ return Symfony\CS\Config\Config::create()
     ->level(\Symfony\CS\FixerInterface::PSR2_LEVEL)
     ->fixers(
         array(
-            'unalign_double_arrow ',
+            'align_double_arrow',
```

Comment:
This conflicts (space was removed), and it should stay as `unalign` imo; aligning creates extra whitespace diff noise :s